lib/interviewer/selectors/name-generator.js

```diff
@@ -23,13 +24,27 @@ const stageCardOptions = (_, props) => props.stage.cardOptions;
 const stageSortOptions = (_, props) => props.stage.sortOptions;
 const propPanels = (_, props) => props.stage.panels;
 
+const getIDs = createSelector(
+  propStageId,
+  getPromptId,
+  (stageId, promptId) => {
+    return {
+      stageId,
+      promptId,
+    };
+  },
+)
+
 export const getPromptModelData = createSelector(
   getStageSubject,
-  getIds,
-  ({ type }, ids) => ({
-    type,
-    ...ids,
-  }),
+  getIDs,
+  ({ type }, { stageId, promptId }) => {
+    return {
+      type,
+      stageId,
+      promptId,
+    };
+  },
 )
 
 export const makeGetPromptNodeModelData = () => getPromptModelData;
```

Comment:
I didn't understand the purpose of this function because it's not being called in anywhere